### PR TITLE
Support composite tiles in PredictiveCacheController

### LINKED ISSUE #4185: Refactor PredictiveCacheController to accommodate for composited tiles
Capturing from @kkaefer and @pozdnyakov that `tilestorefilesource` supports now composited tiles so we should be able to remove this check https://github.com/mapbox/mapbox-navigation-android/blob/845235a327e66a2b380512389[…]java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt and test that everything works as expected.

### PR DESCRIPTION
**Pull request by korshaknn**
### Description

closes #4185

### Changelog

```
<changelog>Support composite tiles in PredictiveCacheController</changelog>
```

### CODE REVIEW

File: libnavigation-base/src/main/java/com/mapbox/navigation/base/options/PredictiveCacheLocationOptions.kt [hunk 1]
```diff
@@ -3,9 +3,9 @@ package com.mapbox.navigation.base.options
 /**
  * PredictiveCacheLocationOptions.
  *
- * @param currentLocationRadiusInMeters How far around the user's location we're going to cache, in meters. Defaults to 20
- * @param routeBufferRadiusInMeters How far around the active route we're going to cache, in meters (if route is set). Defaults to 5
- * @param destinationLocationRadiusInMeters How far around the destination location we're going to cache, in meters (if route is set). Defaults to 50
+ * @param currentLocationRadiusInMeters How far around the user's location we're going to cache, in meters. Defaults to 20000 (20 km)
+ * @param routeBufferRadiusInMeters How far around the active route we're going to cache, in meters (if route is set). Defaults to 5000 (5 km)
+ * @param destinationLocationRadiusInMeters How far around the destination location we're going to cache, in meters (if route is set). Defaults to 50000 (50 km)
```
Comment (LukasPaczos) [inline libnavigation-base/src/main/java/com/mapbox/navigation/base/options/PredictiveCacheLocationOptions.kt:8]: Those changes are incorrect, aren't they?

Comment (LukasPaczos) [reply libnavigation-base/src/main/java/com/mapbox/navigation/base/options/PredictiveCacheLocationOptions.kt:8]: I looked at a diff backwards :facepalm:

Comment (LukasPaczos) [reply libnavigation-base/src/main/java/com/mapbox/navigation/base/options/PredictiveCacheLocationOptions.kt:8]: Thanks for fixing it up!

**Review by LukasPaczos — APPROVED** [review]
lgtm

**Comment by Guardiola31337** [comment]
cc'ing @kkaefer for visibility

**Comment by Guardiola31337** [comment]
In preparation for the release, going ahead and merging here 🚀 

cc @korshaknn

**Pull request merged**